notify.plugin.zsh

```diff
@@ -27,15 +29,21 @@ zstyle ':notify:*' error-sound ''
 zstyle ':notify:*' error-icon ''
 zstyle ':notify:*' disable-urgent no
 zstyle ':notify:*' activate-terminal no
+zstyle ':notify:*' always-check-active-window no
 
 unset plugin_dir
 
 # store command line and start time for later
 function zsh-notify-before-command() {
-    local window_id impl
```

Comment:
ah, this `local` is actually a leftover from my latest changes (sorry!), so `window_id` in this function will always evaluate equal to `""` below.

same for `always_check_wid`, there should be a call to 

    zstyle -s ':notify:*' always-check-active-window always_check_wid

to read the value that might have been set by the user in their `.zshrc`.

Reply:
Yup, I actually did the right thing originally, but then screwed up when I merged. Fixed now.